authors/apps/articles/serializers.py

```diff
@@ -28,6 +30,16 @@ def validate_user_permissions(self, request, data):
                 status=status.HTTP_403_FORBIDDEN
             )
 
+    def article_time_to_read(self, data):
+        """Method to calculate the total time to read an article
+        """
+
+        total_words = data.get('description') + ' ' + data.get('body')
+
+        time_to_read = len(total_words.split()) / 180
```

Comment:
Add a function to strip html tags. This is because we need to calculate the time to read the words only, exclude html tags.

Reply:
@bettblake08  I am working in that will update that method. Thanks for spotting that.

Reply:
@bettblake08 it's now updated, the <> in the body will be removed when estimating time to read an article.